### PR TITLE
FDI-46 directory page

### PR DESCRIPTION
**Pull request by josue2591**
[FDI-46](https://manati.atlassian.net/browse/FDI-46)
-
**Steps to test**
- Ensure the backend is running
- On the back-end side create several directorio nodes, as many as you want at least 9 (to see the design)
- Go to http://localhost:8080/directorio
- There you should be able to see all the directorio nodes and be able to filter them

### CODE REVIEW

File: src/views/Directorio.vue [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  margin-bottom: 20px;
+  color: $text;
+  background: url(.././assets/images/icons/search-icon.svg) no-repeat;
+  background-position: right 10px center;
+  padding-left: 10px;
+  &::-webkit-input-placeholder{
+    font-style: italic;
+    color: #e0e0e0;
+  }
+  &::-moz-placeholder {
```
Comment (cienvaras) [inline src/views/Directorio.vue]: You're not styling the standard form of this selector (`::placeholder`). See: https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder. Also, I'm pretty sure this project has autoprefixer available. That'd mean that the browser-specific versions are not needed. Please check this.

**Comment by cienvaras** [comment]
A couple of changes:
* Content isn't centered, there's more space on the right:
screen shot 2018-04-23 at 8 59 31 am
* Please change URL to `/recursos/directorio/` to respect hierarchy.
* Please add "ver todos" link to directory block in recursos page, linked to this page.

**Comment by cienvaras** [comment]
@josue2591 Did you validate "ver más" link position with Keren or Lu? In the prototype it is in a different position:

screen shot 2018-04-23 at 10 06 31 pm

**Comment by josue2591** [comment]
Yes, I do. I asked Keren because in low resolutions next to the title looks bad

**Comment by cienvaras** [comment]
:shipit:

**Pull request merged**